src/MagnumPlugins/ObjImporter/ObjImporter.cpp

```diff
@@ -53,7 +53,7 @@ void ignoreLine(std::istream& in) {
     in.ignore(std::numeric_limits<std::streamsize>::max(), '\n');
 }
 
-template<std::size_t size> Math::Vector<size, Float> extractFloatData(std::string str, Float* extra = nullptr) {
+template<std::size_t size> Math::Vector<size, Float> extractFloatData(const std::string& str, Float* extra = nullptr) {
```

Comment:
Wow, why am I always doing the stupidest things in the hottest code paths? :D Thanks, this makes the plugin suddenly ∞ times faster as well.